packages/engine-api-client/README.md

```diff
@@ -0,0 +1,68 @@
+# `@lodestar/engine-api-client`
+
+[![Discord](https://img.shields.io/discord/593655374469660673.svg?label=Discord&logo=discord)](https://discord.gg/aMxzVcr)
+[![Eth Consensus Spec v1.1.10](https://img.shields.io/badge/ETH%20consensus--spec-1.1.10-blue)](https://github.com/ethereum/consensus-specs/releases/tag/v1.1.10)
+![ES Version](https://img.shields.io/badge/ES-2020-yellow)
+![Node Version](https://img.shields.io/badge/node-16.x-green)
+
+> This package is part of [ChainSafe's Lodestar](https://lodestar.chainsafe.io) project
+
+Lodestar client for [Engine JSON-RPC API](https://github.com/ethereum/execution-apis/tree/main/src/engine)
+
+> **Note**
+> No version of this package exist prior to version 1.2.1.
+
+
+## Usage
+
+```
+yarn add @lodestar/engine-api-client
+```
+
+Provides method for interacting with the Engine API of the execution layer client as defined in [Engine JSON-RPC API](https://github.com/ethereum/execution-apis/tree/main/src/engine)
+
+```
+ const baseUrl = http://localhost:8551 // running execution layer client
+ const controller = new AbortController()
+ const jwtSecretHex = "" // "jwt secret shared with execution layer client"
+ 
+ const executionEngine = new ExecutionEngineHttp(
+   {
+     urls: [baseUrl],
+     retryAttempts: 1,
+     retryDelay: 2000,
+     queueMaxLength: 2,
+     jwtSecretHex
+   },
+   {signal: controller.signal}
+ );
+ 
+ // Prepare a payload
+ const payloadId = await executionEngine.notifyForkchoiceUpdate(
+  genesisBlockHash,
+  safeBlockHash,
+  finalizedBlockHash,
+  {
+      timestamp: quantityToNum("0x..."),
```

Comment:
User should never have to use this data transformers. This are meant to interact directly with execution layer data which is exactly what this package is abstracting

Reply:
This example was actually copied from the `merge-interop.test.ts` file (see [here](https://github.com/ChainSafe/lodestar/blob/8634572c177ba74d271bc71e1c5a4bb0abc73e71/packages/beacon-node/test/sim/merge-interop.test.ts#L121))

I am not sure why because the API do not actually need those data transformers as the actually value can be set directly. I will update the example to have something like:

```
 // Prepare a payload
 const payloadId = await executionEngine.notifyForkchoiceUpdate(
  ForkName.bellatrix,
  genesisBlockHash,
  safeBlockHash,
  finalizedBlockHash,
  {
      timestamp: 1670268513458, // QUANTITY, integer value as defined in ethereum execution layer JSON RPC https://eth.wiki/json-rpc/API
      prevRandao: new Uint8Array(32), // DATA, 32 Bytes as defined in ethereum execution layer JSON RPC https://eth.wiki/json-rpc/API,
      suggestedFeeRecipient: "0x...",
  }
 );
```

Is that acceptable by you?

Reply:
> Is that acceptable by you?

It's not what's acceptable to me, but what's the most useful for consumers of this API. The goal of the README example is to show a quick and concise example to get a feel of how this library is used in the real world